docs/examples/manifests/user_monitoring/4_create_oadp_alert_rule.yaml

```diff
@@ -0,0 +1,18 @@
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  name: sample-oadp-alert2
+  namespace: openshift-adp
+spec:
+  groups:
+  - name: sample-oadp-backup-alert
+    rules:
+    - alert: OADPBackupFailing
+      annotations:
+        description: 'OADP had {{$value | humanize}} backup failures over the last 2 hours.'
+        summary: OADP has issues creating backups
+      expr: |
+        increase(velero_backup_failure_total{job="openshift-adp-velero-metrics-svc"}[2h]) > 0
+      for: 5m
+      labels:
+        severity: warning
```

Comment:
could we have just one file (with list of separating files with `---`) to apply everything at once?

Reply:
I really want to split in case someone already have user workload monitoring enabled on the cluster. The alerting rule is just an example so don't want this to be all in one. The 3rd one is to add service monitor to the user workload monitoring which may have been enabled already.